test/integration/snapshotDelegationModule.spec.ts

```diff
@@ -0,0 +1,137 @@
+import "module-alias/register";
+
+import { ContractTransaction, Address } from "@utils/types";
+import { Account } from "@utils/test/types";
+import { SetToken, DelegateRegistry, GovernanceModule, SnapshotGovernanceAdapter } from "@utils/contracts";
+import { ADDRESS_ZERO } from "@utils/constants";
+import DeployHelper from "@utils/deploys";
+import { ether } from "@utils/index";
+import {
+  addSnapshotBeforeRestoreAfterEach,
+  getAccounts,
+  getRandomAddress,
+  getWaffleExpect,
+  getSystemFixture,
+} from "@utils/test/index";
+import { SystemFixture } from "@utils/fixtures";
+
+const expect = getWaffleExpect();
+
+describe("SnapshotDelegationModule", () => {
+  let owner: Account;
+  let deployer: DeployHelper;
+  let setup: SystemFixture;
+
+  const ZERO_ID = "0x0000000000000000000000000000000000000000000000000000000000000000";
```

Comment:
This would be a good addition to @utils/constants, and we can import ZERO_BYTES for example